src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final int DAY = 0;
+    public static final int NAME = 1;
+    public static final int HOUR = 2;
+    public static final int MONEY = 3;
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String enter = System.lineSeparator();
+        StringBuilder result = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
+        LocalDate dayFrom = LocalDate.parse(dateFrom,formatter);
+        LocalDate dayTo = LocalDate.parse(dateTo,formatter);
+        LocalDate day;
```

Comment:
What is the reason to declare this var here? Do it inside the loop where you use it)